vignettes/articles/02-prepare-loans-inputs.Rmd

```diff
@@ -98,6 +98,8 @@ run_prep_calculation_loans(
 )
 ```
 
+## Next steps
+
 Regardless of which case is chosen, the intermediate input files are generated
 in the correct format so that the user can directly continue with the main
 calculation of the stress test, as described in vignette ["Run stress test"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/03-run-stress-test.html).
```

Comment:
I think in the other links, you removed the " signs. maybe also do that here for consistency?